### ISSUE TITLE
Ep. 3: Wrong model for batch norm, change function parameters

### ISSUE THREAD

**Issue by ashwinvis** [mimer-ai/deep-learning-intro#14]
- Errorneous instantiation in notebook and text. `WeatherPredictionModel` instead of `WeatherPredictionModelBatchNorm`. See https://learn.mimer-ai.eu/deep-learning-intro/3-monitor-the-model/#batchnorm-the-standard-scaler-for-deep-learning
- `accumulate` is not the best name for storing targets for inference. `store_targets` maybe?
- Arrange order of function parameters in a nicer way and make them consistent across episodes. See `fit` and `fit_with tensorboard`:

```py
def fit(model, train_dl, loss_fn, optimizer, val_dl):
    ...

def fit_with_tensorboard(model, train_loader, val_loader, loss_fn, optimizer):
    ...
```
